tests/pages/services/QuotaOverview-cy.js

```diff
@@ -0,0 +1,156 @@
+describe("Quota Tab", function() {
+  context("No quota set", function() {
+    beforeEach(function() {
+      cy.configureCluster({
+        mesos: "1-task-healthy-with-region",
+        nodeHealth: true
+      });
+    });
+
+    it("Shows the quota tab", function() {
+      cy.visitUrl({ url: "/services/overview" });
+      cy.get(".menu-tabbed-item-label-text")
+        .contains("Quota")
+        .click();
+    });
+
+    it("Shows the no quota message", function() {
+      cy.visitUrl({ url: "/services/quota" });
+      cy.get(".panel-content")
+        .contains("No quota defined")
+        .should("exist");
+    });
+
+    it("Shows a working back to services button", function() {
+      cy.visitUrl({ url: "/services/quota" });
+      cy.get(".button-primary")
+        .contains("Back to Services")
+        .click();
+      cy.get(".service-table").should("exist");
+    });
+  });
+
+  context("Quota set", function() {
+    beforeEach(function() {
+      cy.configureCluster({
+        mesos: "1-task-healthy-with-quota",
+        nodeHealth: true
+      });
+      cy.visitUrl({ url: "/services/quota" });
+    });
+
+    function getFirstRowName(name) {
+      cy.get(".ReactVirtualized__Grid")
+        .eq(2)
+        .children()
+        .first()
+        .children()
+        .first()
+        .contains(name);
```

Comment:
this selector feels very brittle to me. what do you think of using e.g. a `data-`-property next time to evade the need for such selectors?

Reply:
What do you mean with a `data` property?

Reply:
a `data-column=name` on the node we are looking for would make the selector less brittle. we could maybe do something like 

`cy.get(".ReactVirtualized__Grid [data-column=name]").first()` instead of encoding assumptions about the structure of that grid.

Reply:
I agree that the selector is sub-optimal, but our table structure is a bit peculiar. I still don't understand how would `cy.get(".ReactVirtualized__Grid [data-column=name]").first()` work. I don't see a `data-column` property anyway, do you mean that we should add it?

Reply:
yes, that was what i meant to suggest.